modules/tooling-support/src/main/java/org/mule/runtime/module/tooling/internal/DefaultConnectivityTestingService.java

```diff
@@ -58,22 +57,6 @@ public ConnectionValidationResult testConnection(String identifier) {
         throw new MuleRuntimeException(e);
       }
     }
-    Object connectivityTestingObject = temporaryArtifact.getMuleContext().getRegistry().get(identifier);
-    if (connectivityTestingObject == null) {
-      throw new ConnectivityTestingObjectNotFoundException(createStaticMessage("It was not possible to find an object to do connectivity testing"));
-    }
-    Collection<ConnectivityTestingStrategy> connectivityTestingStrategies =
-        temporaryArtifact.getMuleContext().getRegistry().lookupObjects(ConnectivityTestingStrategy.class);
-    for (ConnectivityTestingStrategy connectivityTestingStrategy : connectivityTestingStrategies) {
-      if (connectivityTestingStrategy.accepts(connectivityTestingObject)) {
-        try {
-          return connectivityTestingStrategy.testConnectivity(connectivityTestingObject);
-        } catch (Exception e) {
-          return failure(e.getMessage(), UNKNOWN, e);
-        }
-      }
-    }
-    throw new UnsupportedConnectivityTestingObjectException(createStaticMessage("Could not do connectivity testing over object of type "
-        + connectivityTestingObject.getClass().getName()));
+    return temporaryArtifact.getConnectivityTestingService().testConnection(identifier);
```

Comment:
There is another DefaultConnectivityTestingService in core. Can you find a different name for one of these classes?